packages/react-scripts/template/README.md

```diff
@@ -476,6 +477,52 @@ import { Navbar, Jumbotron, Button } from 'react-bootstrap';
 
 Now you are ready to use the imported React Bootstrap components within your component hierarchy defined in the render method. Here is an example [`App.js`](https://gist.githubusercontent.com/gaearon/85d8c067f6af1e56277c82d19fd4da7b/raw/6158dd991b67284e9fc8d70b9d973efe87659d72/App.js) redone using React Bootstrap.
 
+## Adding Material UI
+
+[Material Design](https://material.io/guidelines/) is an interface design specification create by Google. It's a popular, and increasingly familiar, look and feel for web applications.
+[material-ui](http://www.material-ui.com/) is a library of React components implementing Material Design.
```

Comment:
It is the most popular judging by GitHub stars, it's got over 24,000 - the next-highest that I'm aware of is react-toolbox, which currently has 5,570. There are several libraries that provide Material components, the ones I'm aware of are react-toolbox, react-material, react-md, and muicss. I can't give a good assessment or comparison as I've only used material-ui to any notable extent.